### DISCUSSION TITLE
HTTP status code 100-continue

### DISCUSSION THREAD

**Discussion by stasmir** [projectcontour/contour#5067] [Q&A] [answered]
We are using contour in our k8s clusters. 
And I noticed that if a client sends HTTP header "Expect: 100-continue" to a backend pod then the load balancer automatically responds to client with 100-continue status code before reaching to a pod.

I found an option in Envoy docs:
https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto.html

**proxy_100_continue**
([bool](https://developers.google.com/protocol-buffers/docs/proto#scalar)) If proxy_100_continue is true, Envoy will proxy incoming “Expect: 100-continue” headers upstream, and forward “100 Continue” responses downstream. If this is false or not set, Envoy will instead strip the “Expect: 100-continue” header, and send a “100 Continue” response itself.

Could you please advise if it is possible to pass-through 100-continue code from a client to a pod?

**Comment by skriss** [comment, marked as answer]
@stasmir it's not currently possible to control this HTTP connection manager option through Contour, it's always left at the default value.

If you're interested in being able to toggle this, please open a feature request with some more context on _why_ you need to be able to change this from the non-default setting. Thanks!
